examples/feature_engineering_multiple_contexts/scenario_1/fastapi_server.py

```diff
@@ -0,0 +1,131 @@
+""""
+This is a simple example of a FastAPI server that uses Hamilton on the request
+path to transform the data into features, and then uses a fake model to make
+a prediction.
+
+The assumption here is that you get all the raw data passed in via the request.
+
+Otherwise for aggregation type features, you need to pass in a stored value
+that we have mocked out with `load_invariant_feature_values`.
+"""
+
+import fastapi
+import features
+import named_model_feature_sets
+import pandas as pd
+import pydantic
+
+from hamilton import base
+from hamilton.experimental import h_async
+
+app = fastapi.FastAPI()
+
+# know the model schema somehow.
+model_input_features = named_model_feature_sets.model_x_features
+
+
+def load_invariant_feature_values() -> dict:
+    """This function would load the invariant feature values from a database or file.
+    :return: a dictionary of invariant feature values.
+    """
+    return {
+        "age_mean": 33.0,
+        "age_std_dev": 12.0,
+    }
+
+
+def fake_model(df: pd.DataFrame) -> pd.Series:
+    """Function to simulate a model"""
+    # do some transformation.
+    return df.sum()  # this is nonsensical but provides a single number.
```

Comment:
I think this is distracting cause its so simple here -- a lot of hamilton code is simple and `df.sum()` is going to end up in copy/pastes...